app/controllers/reputation_web_service_controller.rb

```diff
@@ -141,11 +135,20 @@ def client
     @response = @@response
   end
 
+  def get_assignment_id_list(assignment_id_one, assignment_id_two)
+    assignment_id_list = []
+    assignment_id_list << assignment_id_one
+    assignment_id_list << assignment_id_two unless assignment_id_two.zero?
+    assignment_id_list
+  end
+
   def send_post_request
```

Comment:
There is scope of improvement in this method,
1. The URL is hardcoded. Can you find a better place to place this URL in a constant and use it here?
2. This method should only hit the post request, anything else (like generating request body, headers, decrypting response, etc) should not be the scope of this method.